include/ECullingTypes.h

```diff
@@ -22,14 +22,14 @@ enum E_CULLING_TYPE
 };
 
 //! Names for culling type
-const c8 *const AutomaticCullingNames[] =
-		{
-				"false",
-				"box",			  // camera box against node box
-				"frustum_box",	  // camera frustum against node box
-				"frustum_sphere", // camera frustum against node sphere
-				"occ_query",	  // occlusion query
-				0};
+const c8 *const AutomaticCullingNames[] = {
+		"false",
+		"box",			  // camera box against node box
+		"frustum_box",	  // camera frustum against node box
+		"frustum_sphere", // camera frustum against node sphere
+		"occ_query",	  // occlusion query
+		0,
+	};
```

Comment:
why not like this? I think indenting the ending brace is strange
```
const c8 *const foo[] = {
    "bar",
    0,
};
```

Reply:
I'm fine with both. I've chosen this because we usually do two-tab indentation on continuation lines, so this feels natural to me.

Reply:
Let's skip the bikeshedding and keep it.